### DISCUSSION TITLE
New "Convert issue to Discussion" behavior adds noise to open source repos

### DISCUSSION THREAD

**Discussion by rarkins** [community/community#3259] [Discussions]
When "Convert issue to discussion" was created, it was fantastic for our busy open source project. Now it didn't matter if people raised an issue or even aggressive bug report when it was in fact just a config error - we could convert that issue to a support discussion and keep our Issues list for bugs and features.

Now I see that the feature has changed behavior - when an issue is converted to a discussion, the issue *remains* in place, and is closed + locked. So instead of this feature reducing noise in our repository, it's essentially adding noise. Now when people search past issues in our repo it's polluted with closed/locked issues which are often completely misleading (like "Feature X doesn't work at all").

Whomever is the product manager for this, please reconsider it because I don't think you've understood why (some) people use this feature. Perhaps find the open source repos who used it the most and ask them why, and if I'm not the only one doing it this way then consider changing the feature or making it configurable.

**Comment by rarkins** [comment]
By the way, our previous solution for this was to have a dedicated repo for support and we'd *transfer* issues out of the main repo to it whenever it was a config problem and not a feature request or bug. We since archived that support repo in favor of discussions in our primary repo, but this new functionality might make us reverse that decision.

**Comment by evi-liu** [comment]
Hi @rarkins, we are aware that locking is not the ideal state for converted issues. It's an intermediate step right now to ensure no comments are lost in the conversion as we work on fixing a bug where many converted issues lost their comments. Once we fix the bug, we will return to deleting the converted issue.

Apologies for the inconvenience this has caused for your community, and thank you for bearing with us as we work through this non-trivial bug. 

Please see this [thread](https://github.com/github/feedback/discussions/3197) for more details and conversation on this topic.
